mct_quantizers/keras/quantize_wrapper.py

```diff
@@ -128,6 +131,7 @@ def _set_weights_vars(self, is_training: bool = True):
             """
             self._weights_vars = []
             for name, quantizer in self.weights_quantizers.items():
+                name = _weight_name(name)
```

Comment:
no need to handle this here, as you already handled  it in the `init`.
instead fix the name in `add_weights_quantizer` as in `init`